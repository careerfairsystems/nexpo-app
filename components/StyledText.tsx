import * as React from 'react';

import { Text, TextProps } from './Themed';
import { StyleSheet, TextInput, TextStyle } from 'react-native';
import Colors from '../constants/Colors'

export function DefaultText(props: TextProps) {
  return <Text {...props} style={[props.style, { fontFamily: 'montserrat' }]} />;
}

interface textProps {
  text: string,
  style?: TextStyle,
}

export function ArkadText(props: textProps) {
  return <Text style={[styles.text, props.style]}>{props.text}</Text>
}

const styles = StyleSheet.create({
  text: {
    justifyContent: "center",
    textAlign: "center",
    fontFamily: 'montserrat',
    color: Colors.white,
  },
});
