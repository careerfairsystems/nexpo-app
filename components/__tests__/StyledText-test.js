import * as React from "react";
import renderer from "react-test-renderer";

import { DefaultText } from "../StyledText";

it(`renders correctly`, () => {
  const tree = renderer.create(<DefaultText>Snapshot test!</DefaultText>).toJSON();

  expect(tree).toMatchSnapshot();
});
