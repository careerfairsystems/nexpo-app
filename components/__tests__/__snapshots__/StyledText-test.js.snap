// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Array [
        undefined,
        Object {
          "fontFamily": 'main-font-bold',
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
